doc/kernel/other/ring_buffers.rst

```diff
@@ -170,6 +256,47 @@ A data item is removed from a ring buffer by calling
         ...
     }
 
+Bytes data is copied out from a ring buffer by calling
+:cpp:func:`sys_ring_buf_bytes_cpy_get()`.
+
+.. code-block:: c
+
+    u8_t my_data[MY_DATA_BYTES];
+    size_t  ret;
+
+    ret = sys_ring_buf_bytes_cpy_get(&ring_buf, my_data, sizeof(my_data));
+    if (ret != sizeof(my_size)) {
+        /* Less bytes copied. */
+    } else {
+        /* Requested amount of bytes retrieved. */
+        ...
+    }
+
+Data can be retrieved from a ring buffer by direct operations on a ring buffer
```

Comment:
How about:

    Data can be retrieved from a **byte mode** ring buffer by direct
    operations on the ring buffer's memory.  For example: